.gitignore

```diff
@@ -5,3 +5,7 @@ subprojects/*/
 another/
 unit_test_files/
 index
+test.seq/
+unit_test/
+unit_test_2/
+sequences_git/
```

Comment:
Let's not create another random folder with unit test files in the working directory. We can move it all inside the main `unit_test_files` folder.